pkg/kubelet/kubelet_pods.go

```diff
@@ -2131,6 +2144,16 @@ func (kl *Kubelet) convertToAPIContainerStatuses(pod *v1.Pod, podStatus *kubecon
 				}
 				if vol.ReadOnly {
 					rroMode := v1.RecursiveReadOnlyDisabled
+					if b, err := resolveRecursiveReadOnly(vol, supportsRRO); err != nil {
```

Comment:
can `b` be `isRRO` or something? it can still be short, but b confused me at first

Reply:
It is normal to have a single-character variable in Go.
I also feel `isRRO` is more confusing when there are `rroMode` and `supportsRRO`. Just my feeling though.

Reply:
`supportsRRO` is already in use, so let me keep it `b`.
Refactoring can be happen in a follow-up PR if you still like.